fix(docs): 6/5 grammar in `book/` (#2218)
tact-lang/tact: docs/src/content/docs/book/types.mdx

}
```

Learn more about them on the dedicated page: [Structs and Messages][s-n-m].

[s-n-m]: /book/structs-and-messages
[structs]: /book/structs-and-messages#structs
[messages]: /book/structs-and-messages#messages

### Optionals

All [primitive types](#primitive-types), as well as [Structs and Messages](#structs-and-messages), can be nullable and hold a special `null{:tact}` value.

Example of an [optional][optionals]:

```tact
let opt: Int? = null; // Int or null, explicitly assigned null
```

Learn more about them on the dedicated page: [Optionals][optionals].

[optionals]: /book/optionals

### Contracts

> [Contracts](/book/contracts) in Tact serve as the main entry points for smart contracts on TON blockchain. They hold all [functions](/book/functions), [getters](/book/functions#getter-functions), and [receivers](/book/functions#receiver-functions) of a TON contract, and much more.

Reviewer: ```suggestion
[Contracts](/book/contracts) in Tact conveniently represent smart contracts on TON blockchain. They hold all [functions](/book/functions), [getters](/book/functions#getter-functions), and [receivers](/book/functions#receiver-functions) of a TON contract, and much more.
```
